source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -387,6 +424,49 @@ content: |
              }
            }
 
+     .. tab::
+        :tabid: csharp
+
+        .. code-block:: csharp
+
+
+           var client = new MongoClient("mongodb://localhost:27017");
+           // dataKeyId is the UUID from the previous step that was printed
+           // to the console
+           var dataKeyId = "aae4f3b4-91b6-4cef-8867-3113a6dfb27b"
```

Comment:
Should this be something like `"<paste your dataKeyId value here>"`?

Reply:
Yep, good point.